Formula/jmxterm.rb

```diff
@@ -1,16 +1,30 @@
 class Jmxterm < Formula
   desc "Open source, command-line based interactive JMX client"
-  homepage "https://wiki.cyclopsgroup.org/jmxterm/"
-  url "https://downloads.sourceforge.net/project/cyclops-group/jmxterm/1.0.0/jmxterm-1.0.0-uber.jar"
+  homepage "https://docs.cyclopsgroup.org/jmxterm"
+  url "https://github.com/jiaqi/jmxterm/releases/download/v1.0.0/jmxterm-1.0.0-uber.jar"
   sha256 "c1f49b132c435ff5059f48aa560f07feb7a742a9db595ecae7fc34ca0aca054f"
 
+  head do
```

Comment:
I can't see that this a very active project so I would discourage adding this since it should only be used when there is an actual need to test the software. Since the project last release was in 2017 I would encourage upstream to post a new release rather than adding a HEAD spec since it would benefit everyone rather than a small amount of people that actually use `head`

Reply:
That's fair. Beyond adding HEAD, are the other changes acceptable?

Reply:
Yes! Let's update URLs